### PR TITLE
DEV-1313 request subtitles if available.

### PR DESCRIPTION
**Pull request by RudolfDG**
Since both an open OT and closed OT can be available at the same time, I used 2 variables (`openOT_available, closedOT_available`) instead of just something like `ot_type`. `_request_subtitles` contains some duplication because of this. If you see a method to DRY this up, feel free to share :)

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -207,6 +211,27 @@ def _update_metadata(self, fragment, metadata):
...
+        if event.media_type == "video" and event.metadata.closedOT_available:
+            closed_ot_request = generate_make_subtitle_available_request_xml(
+                "closed", event.metadata.media_id, event.metadata.media_id
+            )
+            self.rabbit_client.send_message(
+                self.config["rabbitmq"]["get_subtitles_routing_key"],
+                closed_ot_request,
+                self.config["rabbitmq"]["exchange"],
+            )
+
```
Comment (spacid) [inline app/app.py:234]: To DRY it up:

```suggestion
    def _request_subtitles(self, event):
        if event.media_type == "video":
            subtitle_requests = []
            if event.metadata.openOT_available:
                subtitle_requests.append(
                    generate_make_subtitle_available_request_xml(
                        "open", event.metadata.media_id, event.metadata.media_id
                    )
                )
            if event.metadata.closedOT_available:
                subtitle_requests.append(
                    generate_make_subtitle_available_request_xml(
                        "closed", event.metadata.media_id, event.metadata.media_id
                    )
                )
            for sub_req in subtitle_requests:
                self.rabbit_client.send_message(
                    self.config["rabbitmq"]["get_subtitles_routing_key"],
                    sub_req,
                    self.config["rabbitmq"]["exchange"],
                )
```

File: tests/helpers/test_events_parser.py [hunk 2]
```diff
@@ -86,6 +86,45 @@ def test_parse_metadata_updated_event():
...
+def test_parse_get_metadata_response_with_subtitles():
+    # ARRANGE
+    xml = resources.load_xml_resource("getMetadataResponseWithSubtitles")
+    event_parser = EventParser()
+
+    # ACT
+    event = event_parser.get_event("getMetadataResponse", xml)
+
+    # ASSERT
+    assert event.metadata.openOT_available == True
```
Comment (spacid) [inline tests/helpers/test_events_parser.py:98]: Is there an added benefit for `== True`?

**Review by spacid — APPROVED** [review]

File: tests/helpers/test_events_parser.py (hunk 2 above)
Comment (RudolfDG) [reply tests/helpers/test_events_parser.py:98]: In tests I like the added explicitness, no other benefit.

**Review by maartends — APPROVED** [review]
Ok.
We should, however, try to make the commits smaller. These are the changes for one commit:
```
 app/app.py                                              |  29 +++++++++++++++++++++-
 app/helpers/events_parser.py                            |  40 +++++++++++++++++++++++++++++-
 app/helpers/xml_helper.py                               |  22 +++++++++++++++++
 app/models/metadata.py                                  |  16 +++++++++++-
 app/services/mediahaven.py                              |   1 +
 app/services/rabbit.py                                  |   4 +--
 config.yml                                              |   2 ++
 conftest.py                                             |   6 +----
 tests/helpers/test_events_parser.py                     |  39 +++++++++++++++++++++++++++++
 tests/helpers/test_xml_helper.py                        |   1 +
 tests/resources/getMetadataResponseWithClosedOT.xml     | 173 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 tests/resources/getMetadataResponseWithClosedOTNew.xml  | 184 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 tests/resources/getMetadataResponseWithOpenOT.xml       | 176 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 tests/resources/getMetadataResponseWithOpenOTNew.xml    | 187 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 tests/resources/getMetadataResponseWithSubtitles.xml    | 176 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 tests/resources/getMetadataResponseWithSubtitlesNew.xml | 198 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 tests/test_app.py                                       |   5 +++-
 17 files changed, 1247 insertions(+), 12 deletions(-)
```

**Pull request merged**